src/templates/admin/review/unassign_editor.html

```diff
@@ -35,7 +35,7 @@ <h5>From {{ request.user.full_name }}</h5>
                     &nbsp;&nbsp;
                     <button type="submit" class="button alert" name="skip"><i class="fa fa-step-forward">&nbsp;</i>Skip
                     </button>
-                    <a href="{% url 'review_unassign_editor' article.pk assignment.editor.pk %}" class="button warning" type="submit">Cancel</a>
```

Comment:
I think this doesn't belong here, `review.views.unassign_editor` does require both an `article_id` and an `editor_id`